modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * network_observability/configuring-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-virtualization-config_{context}"]
+= Configuring virtual machine (VM) secondary network interfaces for Network Observability
+You can observe networking patterns on an OpenShift Virtualization setup by identifying eBPF-enriched network flows coming from VMs that are connected to additional network interfaces.
+// NOTE to tech review: 
+// do we need to mention anything about CNI here? I noted this in our convo, but I'm not sure if its relevant in the context of this procedure. Maybe its a prerequisite?  
```

Comment:
The virt docs explicitly mention [OVN-Kubernetes secondary network](https://docs.openshift.com/container-platform/4.16/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.html) so I would suggest to only refer to that in our doc too as it's what we tested :smile_cat:

Reply:
Okay, so I changed the intro sentence to:
You can observe networking patterns on an OpenShift Virtualization setup by identifying eBPF-enriched network flows coming from VMs that are connected to _an Open Virtual Network (OVN)-Kubernetes secondary network._

Reply:
+1